### PR TITLE
Loosen the mixlib-log depedency + misc cleanup

### PR DESCRIPTION
**Pull request by tas50**
1 change per PR. A few misc things while I was in here loosening the dep on mixlib-log.

### CODE REVIEW

**Comment by tas50** [comment]
@lamont-granquist @btm thoughts?

File: lib/chef_zero/endpoints/cookbook_artifacts_endpoint.rb [hunk 1]
```diff
@@ -9,10 +9,10 @@ def get(request)
 
         artifacts = begin
           list_data(request)
```
Comment (btm) [inline lib/chef_zero/endpoints/cookbook_artifacts_endpoint.rb]: was this autocorrected? feels odd that this line wasn't indented past begin as well. but super nit-pick.

Comment (tas50) [reply lib/chef_zero/endpoints/cookbook_artifacts_endpoint.rb]: That wasn't. I can see if it complains when I do it. It is hard to read like this.

Comment (lamont-granquist) [reply lib/chef_zero/endpoints/cookbook_artifacts_endpoint.rb]: yeah i think the reason why this happens is mildly complicated, but i've just been manually fixing them.

i usually throw the begin on a new line and that keeps it all more left justified when it gets autofixed by my editor.

vim finally does support the begin being on the same line and lining up the end statement only indented over twice from the start of the line, which is really how i'd prefer things in an ideal world (and opening brackets on their own line in C code is an abomination).

**Pull request merged**